docs/02-help-desk/miro-guidance.md

```diff
@@ -0,0 +1,30 @@
+# Miro Guidance
+
+This document describes how we manage our Miro Enterprise instance. For official documentation, see the [Miro Help Center](https://help.miro.com/hc/en-us).
+
+## Glossary
+- **Account**: see _Team_.
+- **Board**: a collaborative document.
+- **Board owner**: the person who created the board. This can be changed if needed.
+- **Project**: a subset of boards within a team. It's a way to group boards, like a folder.
+- **Team**: a collection of people and boards within the Bixal Enterprise company. This is also referred to as an "account" in some places in the UI.
+- **Team admin**: a person with special permissions to manage the team.
+- **Team member**: a person who is part of a team, including both Bixal and non-Bixal users.
+
+## Organizing boards
+- Boards are organized by Miro Team.
+- Miro Teams are created for each client or major project.
+- There are also Miro Teams for each practice area.
+- Each Miro Team has a Team Admin. Check with your Project Manager or Practice Area Lead if you need to be added to a team.
+
+## Adding people to boards (Sharing)
+- By default, all people in a Miro Team can see and edit all boards within that team.
+- You can [share boards](https://help.miro.com/hc/en-us/articles/360017730813-Sharing-Boards-and-Inviting-Collaborators) with specific people outside the team. You can also share boards to "anyone with the link."
+- If you want to restrict access to a board, you can remove the default team access using the Share button. Set it to "No Access" or other setting as needed.
+- You can also control access to boards through Projects within Teams.
```

Comment:
Can we clarify Miro team managment UI and MS teams. When we have SSO setup this might be a point of confusion.

Reply:
I did another pass, removing capitalization for words defined in the Glossary section and prefacing first use of word "team" in each sentence with "Miro."